packages/components/tests/dummy/app/templates/components/modal.hbs

```diff
@@ -0,0 +1,669 @@
+      <p>
+        Sets the color scheme for the modal header elements: icon, tagline and title.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">neutral</li>
+        <li>warning</li>
+        <li>critical</li>
+      </ol>
+    </dd>
+    <dt>onOpen <code>function</code></dt>
+    <dd>
+      <p>
+        Callback function invoked when the modal is opened.
+      </p>
+    </dd>
+    <dt>onClose <code>function</code></dt>
+    <dd>
+      <p>
+        Callback function invoked when the modal is closed.
+      </p>
+    </dd>
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
+
+  <h4 class="dummy-h4">Contextual components</h4>
+  <p class="dummy-paragraph" id="contextual-components-modal">The title, the content of the modal dialog, and the
+    actions are passed into the modal as yielded components, using the
+    <code class="dummy-code">Header</code>,
+    <code class="dummy-code">Body</code>,
+    <code class="dummy-code">Footer</code>
+    keys.</p>
+  <dl class="dummy-component-props" aria-labelledby="contextual-components-modal">
+    <dt>&lt;[M].Header&gt; <code>yielded component</code> <strong class="required">required</strong></dt>
+    <dd>
+      <p>It is a container that yields its content as the title of the modal dialog.</p>
+      <p>It accepts
+        <code class="dummy-code">...attributes</code>
+        spreading.</p>
+      <dl class="dummy-component-props">
+        <dt>icon <code>string</code></dt>
+        <dd>
+          <p>Acceptable values: any
+            <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">
+              Flight</a>
+            icon name.</p>
+        </dd>
+        <dt>tagline <code>string</code></dt>
+        <dd>
+          <p>A string that helps the user maintain context when a modal dialog is open.</p>
+        </dd>
+        <dt>...attributes</dt>
+        <dd>
+          <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+        </dd>
+      </dl>
+    </dd>
+    <dt>&lt;[M].Body&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is an unstyled, generic container that yields as the main content of the modal dialog.</p>
+      <p>It accepts
+        <code class="dummy-code">...attributes</code>
+        spreading.</p>
+    </dd>
+    <dt>&lt;[M].Footer&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is a container that yields its content as the footer of the modal dialog.</p>
+      <p>We recommend using it exclusively for actions using the
+        <LinkTo @route="components.button-set"><code class="dummy-code">ButtonSet</code></LinkTo>
+        component. If a tertiary action is presented, it will always be aligned at the end of the row.</p>
+      <p>It accepts
+        <code class="dummy-code">...attributes</code>
+        spreading.</p>
```

Comment:
[non-blocker] this is _really_ strange phrasing to me, but I'm going to go ahead and guess that we've already used it elsewhere so it's "standard" now. Definitely recommend that we change this as soon as is expedient, though.

Reply:
Replaced with 'It supports `...attributes`' – if you have better terminology at hand I'm happy to try a cross-repo update.

Reply:
Not sure what is wrong with `spreading`. It refers to [the `spread` operator in JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax).